code/TestAutomation.Tests/BaseTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using OpenQA.Selenium;
+using TestAutomation.Core.Browser;
+using TestAutomation.Core.Enums;
+using TestAutomation.Core.Utilities;
+using TestAutomation.Epam.PageObjects.Pages;
+using TestAutomation.Utilities;
+namespace TestAutomation.Tests
+{
```

Comment:
No need to create separate layer for BaseTest. Nove it to the TestAutomation.Epam.Tests.